docs/number.adoc

```diff
@@ -33,63 +21,13 @@
 ....
 
 ....
-  call apoc.number.format.pattern.lang(12345, '#,##0.00;(#,##0.00)', 'it') yield value
```

Comment:
don't remove them but replace them with the proper `RETURN apoc.number.format(..` with full parameters